src/components/MerchSection/Merch.tsx

```diff
@@ -12,10 +11,10 @@ const Merch = (props: MerchProps) => {
   return (
     <S.MerchContainer mobileView={!props.desktopView}>
       {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
-      {props.merchItems.map((member: any) => {
+      {props.merchItems.map((member: any, index: number) => {
```

Comment:
Using the index for the element key is not good practice, please use the merch `id` instead